examples/biobert_ner/ner_predict_example.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+
+from termcolor import colored
+
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.readers import StringReader
+from forte.pipeline import Pipeline
+from forte.processors import BERTTokenizer, BioBERTNERPredictor
+from forte.processors.nltk_processors import NLTKSentenceSegmenter
+
+from ft.onto.base_ontology import Subword, Sentence, EntityMention
+
+config = yaml.safe_load(open("config.yml", "r"))
+
+config = Config(config, default_hparams=None)
```

Comment:
I don't see the reason for writing config at the module level. But this is fine for an example class.

Reply:
What is "writing config at the module level"? I was just copying what `examples/pipeline/process_string_example.py` did. What's the best practice for this?

Reply:
The config is at the top level of this module (the current file).  In this example, it can actually be added to the `main`  function. The problem of writing at the module level is that if someone else import this module, then the `config` line will be executed at import time. This is fine since this is just an example script.

Reply:
I see - That makes a lot of sense. I'll be careful about this in the next example script